game.js

```diff
@@ -42,17 +42,14 @@ $(document).keypress(function () {
         nextSequence();
         started = true;
     }
-    else {
-    }
 });
 
 function checkAnswer(currentLevel) {
```

Comment:
This function will not work in this way.

Reply:
Exactly why I came back to the code you created in the beginning.